admin/.env.dist

```diff
@@ -1,4 +1,4 @@
-CONFIG_VERSION=v1.2022-03-18
+CONFIG_VERSION=v10.2022-09-20
```

Comment:
Is this correct this way?
See other occations as well …

Why this is not in the master?
Do I understand something wrong?